challenges.yml

```
@@ -193,3 +193,245 @@ stimulus_values:
         d: |
           `this.values.colorSelected`;
       correct: c
+value_change_callback:
+  question: |
+    Suppose we have a Stimulus controller like this:
+
+    ```js
+    export default class extends Controller {
+        static targets = ['selectItem'];
+
+        static values = {
+            itemId: Number
+        }
+
+        connect() {
+            console.log("Hey I'm connected!");
+        }
+    }
+    ```
+    How would you add a *change callback* method to the `itemId` value?
```

line break above this - between the end of the code block and this sentence.

```suggestion
    How would you add a *change callback* method for the `itemId` value?
```